### DISCUSSION TITLE
Add more precision to the tablet area option

### DISCUSSION THREAD

**Discussion by GH-vanny** [ppy/osu#16235] [Ideas] [answered]
my tablet area is rather precise and i feel different aiming when the tablet widths and heights aren't as precise as mine (example: 34.67, 46.53 etc) are so is it possible to add that? thank you if you consider

**Comment by peppy** [comment, marked as answer]
a difference of 0.47mm should not be discernible, so please try rounding the values for now (35 / 47 should work great in your case)

https://github.com/ppy/osu/discussions?discussions_q=tablet+decimal

**Reply by GH-vanny** [reply]
no problem, thank you
